GitTfsTest/BaseTest.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Sep.Git.Tfs.Test
+{
+    public class BaseTest
+    {
+        public BaseTest()
+        {
+            if (!System.Diagnostics.Debugger.IsAttached)
+            {
+                System.Diagnostics.Trace.Listeners.Clear();
+            }
```

Comment:
Not a big problem for me. If we forget to ingherit from BaseTest, that's just that the build will outpout some logs. Not a big deal and if it is annoying it will be quickly fixed...